debug-bar.php

```diff
@@ -9,18 +9,8 @@
 Text Domain: debug-bar
 */
 
-// If the user is an Automattician (typically a vip_support user), then force-enable Debug Bar.
-add_filter( 'debug_bar_enable', function( $enable ) {
-	if ( is_automattician() ) {
-		return true;
-	}
-
-	if ( defined( 'WP_ENVIRONMENT_TYPE' ) && 'local' === WP_ENVIRONMENT_TYPE ) {
-		return true;
-	}
-
-	return $enable;
-}, PHP_INT_MAX );
+// Disabling as first step to deprecation, see https://lobby.vip.wordpress.com/2022/12/14/deprecation-notice-debug-bar-january-31-2023/
+add_filter( 'debug_bar_enable', '__return_false', PHP_INT_MAX );
```

Comment:
I would leave out this part, just delete our usage of the filter is enough.

Reply:
Haha, you beat me to it, I actually deleted it the second pass 😄